File: src/main/java/org/example/sumNumbersArray/SumArray.java
package org.example.sumNumbersArray;

public class SumArray {
    public static void main(String[] args) {
        System.out.println("Sum Array Element and return its final Sum result!");

        int[] numbers = {2,4,5,6,7,8,10};
        int result = sumArray(numbers);
        System.out.println(result);
    }

    public static int sumArray(int arr[]){
        int sum =0;
        for(int i=0; i <arr.length; i++){
            sum = sum + arr[i];
        }
        return sum;
    }
}

File: src/main/java/org/example/sumNumbersArray/SumArray.js
const sumArray = (numbers) => {
  let sum = 0;
  for (let i = 0; i < numbers.length; i++) {
    sum = sum + numbers[i];
  }
  return sum;
};

let numbers = [2, 4, 5, 6, 7, 8, 10];
let result = sumArray(numbers);
console.log(result);
